salt/fileserver/__init__.py

```diff
@@ -198,11 +198,15 @@ def diff_mtime_map(map1, map2):
     '''
     # check if the mtimes are the same
     if sorted(map1) != sorted(map2):
-        #log.debug('diff_mtime_map: the maps are different')
         return True
 
+    # map1 and map2 are guaranteed to have same keys,
+    # so compare mtimes
+    for filename, mtime in map1.iteritems():
```

Comment:
``iteritems()`` is not python3-compatible. Instead, we need to use ``six.iteritems(map1)`` here.